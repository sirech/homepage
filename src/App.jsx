import React, { Component } from 'react'
import { Helmet } from 'react-helmet'

import logo from './logo.svg'
import './App.css'

class App extends Component {
  render () {
    return (
      <div className='App'>
        <Helmet>
          <title>hceris</title>
        </Helmet>
        <div className='App-header'>
          <img src={logo} className='App-logo' alt='logo' />
          <h2>Welcome to React</h2>
        </div>
        <p className='App-intro'>
          To get started, edit <code>src/App.js</code> and save to reload.
        </p>
      </div>
    )
  }
}

export default App
