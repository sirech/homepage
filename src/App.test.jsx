import React from 'react'
import renderer from 'react-test-renderer'
import App from './App'

describe('components', () => {
  describe('App', () => {
    it('renders correctly', () => {
      const component = renderer.create(<App />)
      expect(component.toJSON()).toMatchSnapshot()
    })
  })
})
