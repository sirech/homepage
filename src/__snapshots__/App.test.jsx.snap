// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components App renders correctly 1`] = `
<div>
  <header
    className="mb-4"
  >
    <nav
      className="navbar bg-faded"
    >
      <a
        className="navbar-brand"
      >
        Mario Fernandez
      </a>
    </nav>
  </header>
  <div
    className="container-fluid"
  >
    <div
      className="card-columns"
    >
      <div
        className="card"
      >
        <div
          className="card-block"
        >
          <h5
            className="mb-0 card-title"
          >
            Links
          </h5>
        </div>
        <ul
          className="list-group list-group-flush"
        >
          <li
            className="list-group-item"
          >
            <a
              href="https://github.com/sirech"
            >
              Github Repositories
            </a>
          </li>
          <li
            className="list-group-item"
          >
            <a
              href="https://stackoverflow.com/users/3785/mario-f"
            >
              Stackoverflow
            </a>
          </li>
        </ul>
      </div>
      <div
        className="card"
      >
        <div
          className="card-block"
        >
          <h5
            className="mb-0 card-title"
          >
            Talks
          </h5>
          <p
            className="card-text"
          >
            I used to hate talking in front of people. But I have done technical presentations for a while, and at some point I started to like it. Not that I have mastered, though. For instance, I still tend to talk too fast. I have mostly done technical presentations, at work and at ruby meetups. Hope to eventually get some experience with bigger crowds
          </p>
        </div>
        <ul
          className="list-group list-group-flush"
        >
          <li
            className="list-group-item"
          >
            <a
              href="https://github.com/sirech/talks/raw/master/2016-02-rubygroupmunich-mediumapps.pdf"
            >
              Ruby at XING
            </a>
          </li>
          <li
            className="list-group-item"
          >
            <a
              href="https://github.com/sirech/talks/raw/master/2016-04-rubygroupmunich-rubocop.pdf"
            >
              Using RuboCop
            </a>
          </li>
          <li
            className="list-group-item"
          >
            <a
              href="https://github.com/sirech/talks/raw/master/2017-03-es2017.pdf"
            >
              Intro to ES2017
            </a>
          </li>
          <li
            className="list-group-item"
          >
            <a
              href="https://github.com/sirech/talks/blob/master/2017-04-rubygroupmunich-yai.pdf"
            >
              Assign PRs automatically with YAI
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
